app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>SHOW INFO OF ONE USER</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <div class='container'>
+      <div class='card'>
+        <img class='userpic' src= <%= @user.photo %> >
+        <div class='leftcard'>
+          <h2 class='username'><%= @user.name %></h2>
+          <p class='count'>Number of posts: <%= @user.posts_counter >= 0 ? @user.posts_counter : '0' %></p><br>
+        </div>
+      </div>
+      <div class='biodiv'>
+        <p>Bio</p>
+        <p><%= @user.bio %></p>
+      </div>
+      <% @posts.each do |post| %>
```

Comment:
- I would advise you to use `#recent_post` method because you need to render the user's most recent post

Reply:
I am using the recent post method, it is being done in the controller. The @posts variable passed to users/show.html.erb is defined as:

`@posts = @user.recent_posts`